test/en_opf_bounds.jl

```diff
@@ -76,7 +76,9 @@ end
         # IVRReducedQuadraticENPowerModel does not implement sm_ub
 
         # ACRENPowerModel
-        sol_pmd = calc_sol_pmd(data_math, ACRENPowerModel; optimizer=optimizer_with_attributes(Ipopt.Optimizer, "mu_strategy"=>"adaptive", "sb"=>"yes", "print_level"=>0))
+        # This test is flakey.
+        mu_strategy = Sys.islinux() ? "adaptive" : "monotone"
+        sol_pmd = calc_sol_pmd(data_math, ACRENPowerModel; optimizer=optimizer_with_attributes(Ipopt.Optimizer, "mu_strategy"=>mu_strategy, "sb"=>"yes", "print_level"=>0))
```

Comment:
@pseudocubic I changed the `mu_strategy` to the default except for linux because this test seemed to fail sporadically. I assume that the non-default option was added precisely because it's required sometimes (but not all the time).